src/librustc/util/ppaux.rs

```diff
@@ -32,6 +32,10 @@ use syntax::ast::CRATE_NODE_ID;
 use syntax::symbol::{Symbol, InternedString};
 use hir;
 
+thread_local! {
```

Comment:
comment please =)

Reply:
Updated most recent commit to address this.